### PR TITLE
Update contributing docs with caveat

### LINKED ISSUE #746: [Docs] Update Layer5 Contributing Doc
**Current Behavior**
The CONTRIBUTING.md is missing the caveat about Jekyll production & dev env amongst others.

**Desired Situation**

[ ] Add [this caveat as seen in line 71](https://github.com/layer5io/service-mesh-performance/commit/a8af02878eb4711ccadb2e4fe01dad7b65a5b1b3#diff-6a3371457528722a734f3c51d9238c13)  to the [CONTRIBUTING.md](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#contribution-flow) Contribution Flow section.

[ ] Update Set up your development environment section as seen in line [49-54](https://github.com/layer5io/service-mesh-performance/commit/a8af02878eb4711ccadb2e4fe01dad7b65a5b1b3#diff-6a3371457528722a734f3c51d9238c13).

---
[Optional] **Alternatives**

[Optional] **Additional context**

---
**Contributor Resources**
The `layer5` repo contains two websites. The current generation and the next-generation of the layer5.io site.

If the `layer5-ng` label is absent on this issue, then this issue pertains to the current generation of the layer5.io website, which uses Jekyll and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [`layer5` contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)

If the `layer5-ng` label is present on this issue, then this issue pertains to the next-generation of the layer5.io website, which uses Gatsby, Strapi, and GitHub Pages. Site content is found under the [`layer5-ng` branch](https://github.com/layer5io/layer5/tree/layer5-ng).
- See [`layer5-ng` contributing instructions](https://github.com/layer5io/layer5/blob/layer5-ng/CONTRIBUTING.md)

### PR DESCRIPTION
**Pull request by Hack-Light**
Signed-off-by: Hack-Light <onohsomtochukwubasil@gmail.com>

**Description**

This PR fixes #746 

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -43,40 +43,53 @@ Signed-off-by: Jane Smith <jane.smith@example.com>
...
-`git checkout -b <my-username>/<my-changes>` 
+   `git checkout -b <my-username>/<my-changes>`
 1. Edit site/add content.
-`vi <specific page>.md` # or use your favorite IDE
+   `vi <specific page>.md` # or use your favorite IDE
 1. Run site locally to preview changes.
-`make site` # this will run a local web server with "live reload" conveniently enabled.
+   `make site` # this will run a local web server with "live reload" conveniently enabled.
+
+- **Note:** _From the Makefile, this command is actually running `$ bundle exec jekyll serve --drafts --livereload`. There are two Jekyll configuration, `jekyll serve` for developing locally and `jekyll build` when deploying for production based on differences of the handling of trailing slashes between Jekyll and GitHub Pages._
```
Comment (PluckyPrecious) [inline CONTRIBUTING.md:89]: Hi, a collaborator reviewed this line in the other repo and requested that it be changed. Hope you don’t mind changing it to this ”There are two Jekyll configuration, `jekyll serve` for developing locally and `jekyll build` when you need to generate the site artefacts for production.”

**Review by PluckyPrecious — COMMENTED** [review]
Well done 👍🏽 @Hack-Light, just a minor tweak and we are good to go🥳. Also the PR title should always be a short description of work done, “Update contributing docs with caveat” isn’t bad what do you think? 🤔

File: CONTRIBUTING.md (hunk 1 above)
Comment (Hack-Light) [reply CONTRIBUTING.md:89]: Ok let me get to it

**Comment by Hack-Light** [comment]
Done

**Review by PluckyPrecious — APPROVED** [review]
Great 👍🏽 @Hack-Light 
@leecalcote PR looks good.🥳

**Review by leecalcote — APPROVED** [review]
Excellent! Good work, @Hack-Light. 👍 

Thank you, @PluckyPrecious

**Pull request merged**